core/variant/dictionary.h

```diff
@@ -74,6 +74,7 @@ class Dictionary {
 
 	int size() const;
 	bool is_empty() const;
+	void reserve(int64_t p_new_capacity);
```

Comment:
```suggestion
	void reserve(int p_new_capacity);
```
Same as with `Array` sizing for this type is `int` elsewhere this creates a potentially confusing mismatch

Reply:
See discussion above: https://github.com/godotengine/godot/pull/110709#discussion_r2365554909

Reply:
Alright, I guess `int` would be better for now since `size()` returns `int` as well. As with `Array`, I plan to change this anyway, but it doesn't have to be now.

Reply:
Changing it from `int32_t` to `int64_t` in the future won't break compatibility so that would be safe to do